docs/usage.rst

```diff
@@ -68,8 +68,8 @@ attribute then you can write a function ``hawk_lookup`` as follows:
 
 .. code-block:: python
 
-    def hawk_lookup(id):
-        user = HawkUser.objects.get(some_id=id)
+    def hawk_lookup(cr_id):
```

Comment:
`cr_id` is a bit cryptic. Can we keep it as `id` for the example code? Also the return value wasn't updated.